custom_components/mawaqit/__init__.py

```diff
@@ -217,6 +215,22 @@ def get_new_prayer_times(self):
         res['Next Salat Name']=salat_name[liste.index(min(liste))]
         #15 minutes before next salat
         res['Next Salat Preparation'] = (datetime.strptime(min(liste), '%Y-%m-%d %H:%M:%S')-timedelta(minutes=15)).strftime('%Y-%m-%d %H:%M:%S').split(" ",1)[1].rsplit(':', 1)[0]
+        
+        # if jumu'a is set as dhuhr, then jumu'a time is the same as dhuhr time
+        if data["jumuaAsDuhr"]:
+            res['Jumua'] = res['Dhuhr']
```

Comment:
here you have to pick the next friday duhr prayer time and not duhr of the current day